vue/components/ui/molecules/filter/filter.vue

```diff
@@ -321,6 +349,15 @@ export const Filter = {
                 return false;
             }
 
+            // in case of an autorefresh you want to get at least
+            // the same amount of items that are currently being shown
+            if (auto) {
+                this.autoRefreshing = true;

```

Comment:
I think there is a case where this isn't set to `false`.

if `auto` and `freshItems` it returtns without setting autoRefreshing to `false` please double check if the logic is right